src/components/RichLayout/Footer.js

```diff
@@ -26,10 +26,21 @@ class Footer extends React.Component {
     }
   }
 
+  getVersion () {
+    // eslint-disable-next-line no-undef
+    if (__DEV__) {
+      // eslint-disable-next-line no-undef
+      return <div className={style.item}>{__VERSION__}</div>
```

Comment:
__VERSION__ only includes a commit hash when the version is not exactly on that hash, right?

Reply:
There needs to be a little bit more visual separation of the disclaimer link and the version. Maybe you can use '|' for that?

Reply:
This is just for debugging so I think visuals are not important here

Reply:
ah yeah, right. :D